docs/core/tools/dotnet-package-add.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: dotnet add package command
-description: The 'dotnet add package' command provides a convenient option to add a NuGet package reference to a project.
+title: dotnet package add command
+description: The 'dotnet package add' command provides a convenient option to add a NuGet package reference to a project.
 ms.date: 04/13/2022
 ---
-# dotnet add package
+# dotnet package add
 
 **This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
```

Comment:
@Forgind Should we add a note that this only works in .NET 10+ SDK?

Reply:
Good idea!